config/locales/en.yml

```diff
@@ -1995,6 +1995,12 @@ en:
           Resources Canada), and StatCan (Geography Division,
           Statistics Canada).
         contributors_ca_canada: Canada
+        contributors_cz_credit_html: |
+          %{czechia}: Contains data from the State Administration of Land Surveying
+          and Cadastre licensed under ${cc_licence_link}
```

Comment:
There's a typo here: it should be `%{cc_licence_link}` (i.e. `%` not `$`)

Reply:
OK, one more try ;-) Thanks.

Reply:
Is there anything more I can do for this on my side please?